src/gui/gtk.c

```diff
@@ -1092,6 +1098,10 @@ int dt_gui_gtk_init(dt_gui_gtk_t *gui)
   dt_accel_connect_global("switch view",
                           g_cclosure_new(G_CALLBACK(view_switch_key_accel_callback), NULL, NULL));
 
+  // Global zoom in & zoom out
+  dt_accel_register_global(NC_("accel", "zoom in"), GDK_KEY_KP_Add, GDK_CONTROL_MASK);
```

Comment:
Does that also work with normal `+` and `-` keys or just the key pad ones?

Reply:
No, the default just works with the keypad ones, I believe that's the most common to use case, as many keyboard layouts put + and - accessible only with a SHIFT key, ie the US QUERTY keyboard, so using that for a common task like zooming would require an octopus...

The good news is that you can redefine these accelerators to simple + and - keys, I gave a try with my German keyboard and it was working (on Gemran keyboard these keys do not require SHIFT). BTW Interestingly when you change a global accelerator it only works if you restart dt?

Reply:
Well, I don't even have a numblock. So I would prefer to see normal +/- being used. That's also what other applications like web browsers default to.

Reply:
Fair enough, let me change it by default to `GDK_KEY_plus` and `GDK_KEY_minus`.